server/public/model/compliance.go

```diff
@@ -6,6 +6,8 @@ package model
 import (
 	"net/http"
 	"strings"
+
+	"github.com/mattermost/mattermost/server/public/shared/mlog"
```

Comment:
This should also be added to the `go.mod` for public/model package right?

Reply:
Sorry, my bad. Just confused from the shared/mlog. For a moment thought we have a public module for `model` only.